test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -203,22 +163,25 @@ func TestNamespaceScheduler(t *testing.T) {
 
 			clients, err := clientset.NewClusterForConfig(cfg)
 			require.NoError(t, err, "failed to construct client for server")
-			clusterClient := clients.Cluster(clusterName).WorkloadV1alpha1().WorkloadClusters()
+			kcpClusterClient := clients.Cluster(clusterName)
```

Comment:
no, this is no cluster client.

Reply:
What would you like to see it changed to?

Reply:
Done.